include/swift/Runtime/Metadata.h

```diff
@@ -1233,17 +1230,14 @@ struct TargetNominalTypeDescriptor final
       /// properties in its metadata, if any. 0 means there is no field offset
       /// vector.
       uint32_t FieldOffsetVectorOffset;
-      
-      /// The field names. A doubly-null-terminated list of strings, whose
-      /// length and order is consistent with that of the field offset vector.
+
+    private:
+      // FIXME: This is left as a placeholder (which is always 0) because
```

Comment:
I thought GenReflection.cpp emits enum case names in the field names section too. The runtime mirrors implementation should use that, and we should remove this field entirely.

Reply:
I think we can change the NominalTypeDescriptor layout as a separate step once the implementation no longer needs these fields. Pavel tried removing the fields and ran into other problems where the layout was depended on. Hopefully I shook those out since I just changed the format of everything in #13468, but it still doesn't hurt to make incremental steps.